### DISCUSSION TITLE
ChicOrtiz: Ausencia nos dias 07 e 21 de Janeiro de 2022

### DISCUSSION THREAD

**Discussion by chicortiz** [clube-de-leitura-python/pense-em-python-2021#35] [Conversa geral]
Pessoal! Feliz ano novo a todos! Me perdoem as ausências, os encontros estão maravilhosos, continuem por favor!!! Dia 07 eu estava viajando e no dia 21 terei médico, sinto muito. Muita saudade de todos vocês! Espero reencontra-los mais à frente :)

**Comment by villares** [comment]
Oba gente, Chico, @omyasuda e @claudewick,
Queria deixar aqui as minhas desculpas! Dei mancada novamente!!! Anotei errado na agenda e furei o último encontro, perdão.. tenho compromisso esta sexta, vou marcar o próximo 4/2, e re-convidar o pessoal.

**Reply by villares** [reply]
errata: marquei lá na página o próximo 4/2
